content/articles/i-got-an-ecc-certificate-but-i-want-a-rsa-one.markdown

```diff
@@ -1,58 +1,34 @@
 ---
 title: I got an ECC-signed certificate but want RSA
-excerpt: How to reissue an SSL Certificate using an RSA Signing Key.
+excerpt: What to do when you need your SSL Certificate to be signed with an RSA key.
 categories:
 - SSL Certificates
 ---
 
-# Reissuing an SSL Certificate using an RSA Signing Key
+# I got an ECC-signed certificate but want RSA
 
-DNSimple provides SSL certificates using elliptic curve keys by default, but there are some situations or requirements where you still need to use an RSA key as the certificate signing key.
+DNSimple provides SSL certificates using elliptic curve keys by default, but there are some situations or requirements where you still need to use an RSA key as the certificate signing key. 
 
-The RSA-signed certificates are available by generating a custom Certificate Signing Request (CSR) and using it while reissuing the SSL certificate.
+The following sections explain what you need to do to get your SSL certificates signed with RSA keys.
 
-## Before starting
+## Let'sEncrypt certificates
 
-In this guide, we assume you have OpenSSL installed in your system. If you don't have it available, you will need to install it:
+SSL certificates issued by Let'sEncrypt can't be reissued, which means that you will need to order a new certificate or manually renew an existing one. Please, follow these steps:
 
-* Using your favorite package manager: [apt](https://packages.debian.org/bullseye/openssl), [dnf](https://packages.fedoraproject.org/pkgs/openssl/openssl/), [brew](https://formulae.brew.sh/formula/openssl@3), [nuget](https://www.nuget.org/packages/open_ssl), [chocolatey](https://community.chocolatey.org/packages/openssl), etc.
-* Using the [OpenSSL binaries](https://wiki.openssl.org/index.php/Binaries).
-* Finding other alternatives to generate the RSA private key and the CSR.
+1. Ensure auto-renewal is disabled on the certificate you want to replace.
```

Comment:
I recommend to add an extra statement to highlight the value of this step. I anticipate several customers just skipping it for "conveniency", as we issue certificates "for free".

We can expand by stating that this is very important to avoid the other certificate to continue to be issued, and consume their Let's Encrypt quota possibly resulting in issuance requests to be throttled or denied.

Reply:
Good point, @weppos. However, we already have a validation that will prevent a customer from creating a new certificate if there's another one with auto-renewal enabled for the same set of names.

The new step in those instructions is to ensure the customer doesn't get any errors and can go through the process without any surprises.

Reply:
I was not aware of that validation. If you confirm it exists, it makes less relevant this addition.